browsers.config.js

```diff
@@ -8,14 +8,13 @@ const defaultBrowsers = ['chrome', 'firefox'];
 const availableBrowsers = ['chrome', 'firefox', 'opera'];
 
 // BrowserStack browsers
-const defaultBSBrowsers = ['chrome', 'firefox', 'edge'];
-const previousBSBrowser = ['chrome_previous', 'firefox_previous', 'edge_previous'];
-const supportedBSBrowsers = [...defaultBSBrowsers, ...previousBSBrowser];
+const defaultBSBrowsers = ['chrome', 'firefox', 'safari'];
```

Comment:
Testing on local and on remote are in different environment. Local no Safari, remote with safari. This is going to make the development workflow confusing.

Reply:
It is a minor problem. I will contact the network to make the cli test command work on the local machine, Currently, I have already created the new workflow which run test on `BrowserStack` manually but still in progress.

Reply:
Fyi, Karma has known issue with Safari's test driver. I'm not really sure if this issue will be fixed anytime soon. I recall this issue back in 5 years ago and it's still the issue today.